testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestGrantRevoke.java

```diff
@@ -111,7 +114,7 @@ public void cleanupRoles()
     public void testGrantRevoke()
     {
         // test GRANT
-        aliceExecutor.executeQuery(format("GRANT SELECT ON %s TO bob WITH GRANT OPTION", tableName));
+        aliceExecutor.executeQuery(format("GRANT SELECT ON %s TO bob", tableName));
```

Comment:
Please add a tests where you first do `GRANT SELECT` and then `GRANT SELECT ... WITH GRANT OPTION`.